src/resources/filters/layout/docx.lua

```diff
@@ -110,11 +110,11 @@ end
 
 function docxAlign(align)
   if align == "left" then
-    return "start"
+    return "let"
```

Comment:
Hello,
it just seems, that it must spell "left" instead of "let"...
But it shouldn't pass a validator, if it was wrong... Just to check.
Thanks